lib/service/feed_service.dart

```diff
@@ -168,6 +173,8 @@ class FeedServiceImpl extends FeedService {
     final service = injector<FeedApi>();
     final isTestnet = Environment.appTestnetConfig;
     final feedData = await service.getFeeds(isTestnet, 10, null, null);
+    hasFeed.value = feedData.events.isNotEmpty;
```

Comment:
Looks like you are missing the cache step. when first `hasFeed` is true, please cache the value in `ConfigurationService`. So when opening the app again, check that flag, the feed will automatically show.